monkey/monkey_island/cc/ui/src/components/ui-components/utils/useInterval.tsx

```diff
@@ -0,0 +1,23 @@
+import {useEffect, useRef} from 'react'
+
+export const useInterval = (callback: () => void, delay: number | null) => {
```

Comment:
I don't think we should or need to override this.

Reply:
what do you mean?

Reply:
I don't understand why we need to wrap the `useInterval` and what this wrapping achieves

Reply:
a general hook to use interval with self-cleaning.